test/cql-pytest/random_util.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020-present ScyllaDB
```

Comment:
You copied a few too many lines verbatim from util.py.

Reply:
fixed